dumpling/export/config.go

```diff
@@ -133,6 +133,7 @@ type Config struct {
 	SessionParams      map[string]interface{}
 	Labels             prometheus.Labels `json:"-"`
 	Tables             DatabaseTables
+	Flag               bool
```

Comment:
Use a more precise appellation to name this variable. What's more, we can just use an unexported variable if it won't be used by other packages.
```suggestion
	specifiedTables bool
```